src/main/java/raisetech/student/manegement/repository/StudentRepository.java

```diff
@@ -7,16 +7,28 @@
 import org.apache.ibatis.annotations.Select;
 import raisetech.student.manegement.data.Student;
 
+/**
+ * 生徒情報に関するデータアクセスを提供するリポジトリインターフェース
+ * 概要：studentsテーブルへのクエリを定義する
+ */
+//生徒情報用リポジトリ
 @Mapper
-//データベースを操作するクラス
 public interface StudentRepository {
 
-  //すべての生徒情報を取得
+  /**
+   * すべての生徒情報を取得するリポジトリメソッド
+   * @return 全生徒情報のリスト
+   */
   @Select("SELECT * FROM students")
   List<Student> searchAllStudents();
 
-  //指定した年齢範囲の生徒情報を取得
-  @Select("SELECT * FROM students WHERE age BETWEEN #{minAge} AND #{maxAge}")
+
+  /**
+   * 指定した年齢範囲で生徒情報を取得するリポジトリメソッド
+   * @param minAge 最小年齢（を含む）
+   * @param maxAge 最大年齢（を含む）
+   * @return 指定年齢範囲に該当する生徒情報のリスト
+   */  @Select("SELECT * FROM students WHERE age BETWEEN #{minAge} AND #{maxAge}")
```

Comment:
細かいですけど、他の箇所とフォーマットを揃えるとこうでしょうか。
```suggestion
   */ 
   @Select("SELECT * FROM students WHERE age BETWEEN #{minAge} AND #{maxAge}")
```
searchStudentByAgeの引数部分のフォーマットも崩れていますし、全体的にコードフォーマットを実施しましょう。

Reply:
`IntelliJ コードフォーマット`としらべるとショートカットキーが出てくるかと思います。
もっというと、セーブ時に自動でフォーマットを行う機能があるので、設定をオンにしておきましょう。

Reply:
> 細かいですけど、他の箇所とフォーマットを揃えるとこうでしょうか。
> searchStudentByAgeの引数部分のフォーマットも崩れていますし、全体的にコードフォーマットを実施しましょう。
> IntelliJ コードフォーマットとしらべるとショートカットキーが出てくるかと思います。
> もっというと、セーブ時に自動でフォーマットを行う機能があるので、設定をオンにしておきましょう。


▼上記、レビュー部分について以下のコメント（返答）で回答
https://github.com/kotohogi5/StudentManagement/pull/11#discussion_r2072784366

Reply:
【履歴】対応済み
https://github.com/kotohogi5/StudentManagement/pull/11#discussion_r2072893246